Formula/infrakit.rb

```diff
@@ -0,0 +1,30 @@
+class Infrakit < Formula
+  desc "Toolkit for creating and managing declarative infrastructure"
+  homepage "https://github.com/docker/infrakit"
+  url "https://github.com/docker/infrakit.git",
+      :tag => "v0.5",
+      :revision => "3d2670e484176ce474d4b3d171994ceea7054c02"
+  head "https://github.com/docker/infrakit.git"
+
+  depends_on "automake" => :build
```

Comment:
Are we sure this dep is needed? It feels unlikely.

Reply:
Ops thanks for catching that.